R/begin.R

```diff
@@ -20,7 +21,8 @@ begin = function(
   keep = NULL,
   scipen = FALSE,
   verbose = TRUE,
-  repos = 'http://cran.us.r-project.org'
+  repos = 'http://cran.us.r-project.org',
+  runpath = FALSE
```

Comment:
My preference is runpath = NULL and then check via !is.null(runpath). This is cleaner and consistent with keep. Please use runpath = NULL.